app/assets/stylesheets/index.scss

```diff
@@ -48,27 +75,47 @@
 }
 
 .topic-image {
+  margin-right: 10px;
   width: 20.1px;
   height: 19.5px;
   object-fit: contain;
   display: inline-block;
 }
-.list-topics li {
-  padding-left: 100px;
-  margin-top: 10px;
+
+.topic-list li {
+  margin-top: 15px;
+}
+
+.table-topic-list {
+  font-size: 13px;
+  letter-spacing: 2px;
+  display: table;
+  margin: 0 auto;
+}
+
+.topic-list {
+  list-style-type: none;
 }
 
 .create-first-target {
+  margin-left: auto;
```

Comment:
you can use short version margin: 72px auto 80px auto.   This is (top,right, bottom, left)